README.md

```diff
@@ -23,6 +27,24 @@ This implemented algorithm converts input simplex-wise zigzag filtration to a **
 
 Computation of the standard (non-zigzag) persistence is done by invoking the [phat](https://github.com/blazs/phat) library.
 
+## Using Python Version
+We recommend using anaconda virtual environment. Start by creating a conda environment. We recommend using python >= 3.9.
```

Comment:
Is using py39 really needed? Some may be lazy to not want to upgrade their python (e.g., me), some may still be using py27 (though it should be rare)

Reply:
I think py27 is obsolete, in general Python 2 is. We have to use py37 or later to wrap the C++ code. I think even newer versions of NumPy does not go beyond 3.9. So I am assuming python 3.7 is a safe choice who regularly uses Python.

Reply:
Okay, sounds good.